docs/keyvault.md

```diff
@@ -0,0 +1,31 @@
+# Key Vault plugin for Azure Service Bus
+
+The Key Vault plugin for Azure Service Bus allows for the message body to be encrypted and decrypted by registering a plugin and its accompanying settings.
+
+## How to use
+
+In order to use this plugin you will need to setup the following:
+
+1. An Azure subscription
+
+1. A Service Bus namespace
```

Comment:
No need. Remove the empty lines in between to have a proper numbered list GH will respect.